src/main/kotlin/org/opensearch/commons/alerting/action/IndexCommentResponse.kt

```diff
@@ -0,0 +1,58 @@
+package org.opensearch.commons.alerting.action
+
+import org.opensearch.commons.alerting.model.Comment
+import org.opensearch.commons.alerting.util.IndexUtils.Companion._ID
+import org.opensearch.commons.alerting.util.IndexUtils.Companion._PRIMARY_TERM
+import org.opensearch.commons.alerting.util.IndexUtils.Companion._SEQ_NO
+import org.opensearch.commons.notifications.action.BaseResponse
+import org.opensearch.core.common.io.stream.StreamInput
+import org.opensearch.core.common.io.stream.StreamOutput
+import org.opensearch.core.xcontent.ToXContent
+import org.opensearch.core.xcontent.XContentBuilder
+import java.io.IOException
+
+class IndexCommentResponse : BaseResponse {
+    // TODO: do we really need sequence num and primary term? probs delete em
```

Comment:
I'm also curious. Did you see these variables in a similar API pattern?

Reply:
This pattern is used in Indexing Monitors and are stored here so they can be passed into an OpenSearch IndexRequest. Will take a look at the necessity of this later and determine whether to remove this in a later PR.